api/api.yml

```diff
@@ -0,0 +1,1572 @@
+                       Description: Reserved for national use
+                     '117':
+                       Describe: ans...999
+                       Description: Reserved for national use
+                     '118':
+                       Describe: ans...999
+                       Description: Reserved for national use
+                     '119':
+                       Describe: ans...999
+                       Description: Reserved for national use
+                     '120':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '121':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '122':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '123':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '124':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '125':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '126':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '127':
+                       Describe: ans...999
+                       Description: Reserved for private use
+                     '128':
+                       Describe: b 64
+                       Description: Message authentication code
+                    encoding:
+                     mti_enc: CHAR
+                     bmp_enc: HEX
+                     len_enc: CHAR
+                     num_enc: CHAR
+                     chr_enc: ASCII
+                     bin_enc: HEX
+                     trk_enc: EBCDIC
+            encoding:
+              file:
+                contentType: text/plain
+      responses:
+        '200':
+          description: successful operation
+          content:
+            application/octet-stream:
+              schema:
+                type: string
+                description: iso8583 message
+                format: binary
+                example: '0400F23A400108418202000000400000000019111111111000000000018000000000003000001090806461003310011200909080996010060002000000000003430003948 03808110012000004096565733200000003000001360030003317000394809080646510000000003132020000331609080645190000000020000000000000'
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ISOMessage'
+            application/xml:
+              schema:
+                $ref: '#/components/schemas/ISOMessage'
+        '400':
+          description: bad request
+          content:
+            text/plain:
+              schema:
+                type: string
```

Comment:
I think we'll want to use `.Problem(err)` from moov-io/base to render a JSON object with the error. Thoughts? 

https://github.com/moov-io/base/blob/master/http/server.go#L27

Reply:
fixed